pkgs/os-specific/linux/kernel/zen-kernels.nix

```diff
@@ -1,16 +1,19 @@
 { lib, fetchFromGitHub, buildLinux, ... } @ args:
 
 let
+  # comments with variant added for update script
+  # ./update-zen.py zen
   zenVariant = {
-    version = "5.17";
-    suffix = "zen1";
-    sha256 = "sha256-8DCcO2oPnwXjjm3PW7vNZKbB3S7XsEB/svAMhbPRJIo=";
+    version = "5.17"; #zen
```

Comment:
This won't work in `modDirVersion` for `.0` patch versions. On these, zen tag/version is e.g. `5.17-zen1` (version will be detected as `5.17`), but [`actualModDirVersion` here](https://github.com/NixOS/nixpkgs/blob/80d8655c15055472f0118a877351085cc22c1e92/pkgs/os-specific/linux/kernel/manual-config.nix#L174-L177) will expect `5.17.0` and will error. That's why I have set version as the full `5.17.0-zen1` on the last update and special-cased `rev` to be the `v5.17-zen1` tag on this case.

Reply:
Fixed